src/index.js

```diff
@@ -18,11 +19,23 @@ const todoList = [{
 },
 ];
 
-todoList.forEach(({ description }) => {
+if (!localStorage.getItem('todo')) {
+  localStorage.setItem('todo', JSON.stringify(todoList));
+}
+const todos = JSON.parse(localStorage.getItem('todo') || '[]');
+todos.forEach(({ description, id, completed }) => {
   const div = document.createElement('div');
   div.classList.add('d-height');
   div.classList.add('todo-cont');
-  div.innerHTML = `<li><input type="checkbox" class="check-box"   id="check">${description}</li>
+  div.innerHTML = `<li><input type="checkbox" ${completed && 'checked'} class="check-box"   id="check-${id}">${description}</li>
     <div class="delete-icon"></div>`;
   data.appendChild(div);
-});
\ No newline at end of file
+});
+
+for (let i = 0; i < todos.length; i += 1) {
+  const inCheck = document.querySelector(`#check-${todos[i].id}`);
+  inCheck.addEventListener('input', (e) => {
+    checkItem(i, e, todos);
```

Comment:
- [ ] Although the `input` event handler is working fine you are required to use the `change` event. Those two behave differently, for example, the change event is not necessarily fired for each alteration to an element's value. You can learn more about [here](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event)